packages/js/src/Modules/Verto/BaseSession.ts

```diff
@@ -257,36 +226,7 @@ export default abstract class BaseSession {
    * Callback when the ws connection is open
    * @return void
    */
-  protected async _onSocketOpen() {
-    this._idle = false;
-    const tokenKey = this._jwtAuth ? 'jwt_token' : 'token';
-    const { project, token } = this.options;
-    const bc = new Connect({ project, [tokenKey]: token }, this.sessionid);
-    const response: IBladeConnectResult = await this.execute(bc).catch(
-      this._handleLoginError
-    );
-    if (response) {
-      this._autoReconnect = true;
-      const {
-        sessionid,
-        nodeid,
-        master_nodeid,
-        authorization: { expires_at = null, signature = null } = {},
-      } = response;
-      this.expiresAt = +expires_at || 0;
-      this.signature = signature;
-      this.relayProtocol = await Setup(this);
-      this._checkTokenExpiration();
-      this.sessionid = sessionid;
-      this.nodeid = nodeid;
-      this.master_nodeid = master_nodeid;
-      this._emptyExecuteQueues();
-      this._pong = null;
-      this._keepAlive();
-      trigger(SwEvent.Ready, this, this.uuid);
-      logger.info('Session Ready!');
-    }
-  }
+  protected async _onSocketOpen() {}
```

Comment:
Are you going to remove these here as well?

Reply:
No, I am not. This one is used to register the SocketOpen that comes from his child Verto.Ts